### ISSUE TITLE
Loot ~/.aws/credentials

### ISSUE THREAD

**Issue by jhart-r7** [rapid7/metasploit-framework#6535]
On Linux, OS X and other systems that install AWS' `aws` cli, to use it you need to configure it with one or more profiles, each profile specifying some authentication material that allows `aws` to authenticate to AWS.  The format is like this:

```
[default]
aws_access_key_id - The AWS access key part of your credentials (20 character alphanumeric)
aws_secret_access_key - The AWS secret access key part of your credentials (base64)
aws_session_token - The session token part of your credentials (session tokens only) (base64)
aws_security_token = base64 data for the current session?
[other_profile_name]
aws_access_key_id - The AWS access key part of your credentials (20 character alphanumeric)
aws_secret_access_key - The AWS secret access key part of your credentials (base64)
aws_session_token - The session token part of your credentials (session tokens only) (base64)
aws_security_token = base64 data for the current session?
```

Obtaining `aws_access_key_id` and `aws_secret_access_key` allows anyone with this data to obtain access to the AWS account of whoever owns `aws_access_key_id`.

**Comment by wchen-r7** [comment]
Hi @jhart-r7, could you please provide a brief description? Thanks.

**Comment by james-otten** [comment]
I believe this is covered by `post/multi/gather/aws_keys`

**Comment by jhart-r7** [comment]
Indeed.  I filed the issue and submitted the PR but forgot the paperwork.  Thanks, @james-otten !

**Comment by jhart-r7** [comment]
Closed via #7506
